tests/end-to-end/e2e/e2e_test.go

```diff
@@ -0,0 +1,148 @@
+package e2e
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+
+	"github.com/kyma-incubator/compass/tests/end-to-end/pkg/common"
+
+	"github.com/pkg/errors"
+
+	"github.com/kyma-incubator/compass/tests/end-to-end/pkg/idtokenprovider"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
```

Comment:
Do you really need this init here? Maybe it would be enough to call NewTextContext func on the first lines of `TestCompassAuth`. Also, I don't like the concept of globally shared test context via `tc` variable. In some functions you pass this context as a function parameter, other functions call this `tc` directly.